sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/models/BoundingBox.java

```diff
@@ -1,71 +1,118 @@
+     */
+    public LatLong getTopLeft() {
+        return this.topLeft;
+    }
 
     /**
-     * Get the southWest property: Minimum coordinates (south-west point) of bounding box in latitude longitude
-     * coordinate system. E.g. 52.41064,4.84228.
-     *
-     * @return the southWest value.
+     * Returns the bottom right corner.
+     * @return
      */
-    public List<Double> getSouthWest() {
-        return this.southWest;
+    public LatLong getBottomRight() {
+        return this.bottomRight;
     }
 
     /**
-     * Set the southWest property: Minimum coordinates (south-west point) of bounding box in latitude longitude
-     * coordinate system. E.g. 52.41064,4.84228.
-     *
-     * @param southWest the southWest value to set.
-     * @return the BoundingBox object itself.
+     * Returns the top right corner.
+     * @return
      */
-    public BoundingBox setSouthWest(List<Double> southWest) {
-        this.southWest = southWest;
-        return this;
+    public LatLong getTopRight() {
+        return new LatLong(this.topLeft.getLat(), this.bottomRight.getLon());
     }
 
     /**
-     * Get the northEast property: Maximum coordinates (north-east point) of bounding box in latitude longitude
-     * coordinate system. E.g. 52.41064,4.84228.
-     *
-     * @return the northEast value.
+     * Returns the bottom left corner.
+     * @return
      */
-    public List<Double> getNorthEast() {
-        return this.northEast;
+    public LatLong getBottomLeft() {
+        return new LatLong(this.bottomRight.getLat(), this.topLeft.getLon());
     }
 
     /**
-     * Set the northEast property: Maximum coordinates (north-east point) of bounding box in latitude longitude
-     * coordinate system. E.g. 52.41064,4.84228.
-     *
-     * @param northEast the northEast value to set.
-     * @return the BoundingBox object itself.
+     * Returns the southwest corner
+     * @return
      */
-    public BoundingBox setNorthEast(List<Double> northEast) {
-        this.northEast = northEast;
-        return this;
+    public LatLong getSouthWest() {
+        return getBottomLeft();
+    }
+
+    /**
+     * Returns the northeast corner.
+     * @return
+     */
+    public LatLong getNorthEast() {
+        return getTopRight();
+    }
+
+    BoundingBox(List<Float> bounds) {
+        Objects.requireNonNull(bounds, "Internal bounding box model is needed.");
+        Objects.requireNonNull(bounds.get(0), "Left bound is needed");
+        Objects.requireNonNull(bounds.get(1), "Bottom bound is needed");
+        Objects.requireNonNull(bounds.get(2), "Right bound is needed");
+        Objects.requireNonNull(bounds.get(3), "Top bound is needed");
+
+        final float leftBound = bounds.get(0);
+        final float bottomBound = bounds.get(1);
+        final float rightBound = bounds.get(2);
+        final float topBound = bounds.get(3);
+
+        final String northEastString = String.valueOf(rightBound) + "," + String.valueOf(topBound);
```

Comment:
I don't understand this. 

bounds[0], bounds[1] can be passed directly ot latlong constructor
bounds[2], bounds[3] as well

in fact there is a constructor just for that. they are already double values, no need to convert to string